### DISCUSSION TITLE
Lesson 7 Chainlink AggregatorV3Interface import not working

### DISCUSSION THREAD

**Discussion by AbigirlMuchineripi** [Cyfrin/foundry-full-course-cu#1784] [Q&A] [answered]
Kindly assist I have been stuck here.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/94177156/fc5915f3-cca6-4d8e-8e58-db8777ca58ae)

**Comment by EngrPips** [comment]
Hello @AbigirlMuchineripi, Please manually search for the file. You can use `cltr + p` since you are on Windows. When you get the file's location, you can either do native importing by explicitly declaring the path to the file, or you can use the correct path to create a working mapping. This problem usually happens because the path to the file has changed.

**Comment by snowieoxs** [comment, marked as answer]
```
forge install smartcontractkit/chainlink-brownie-contracts@1.1.1 --no-commit
```

In my foundry.toml:

```
remappings = ['@chainlink/contracts/=lib/chainlink-brownie-contracts/contracts/']
```

I did look into using the contracts from the Chainlink repo... but ended up with using what was in the video, maybe i should have used the Chainlink repo... anyways, toml mappings, have been weird for me... sometimes the console shows you an error of where it looked, helped me fix my mappings

**Reply by EngrPips** [reply]
This works well for you, right?

**Reply by tesla809** [reply]
I am having the same issues. Any word on if this works?

**Reply by tesla809** [reply]
It works. Thanks.

**Reply by EngrPips** [reply]
Awesome.

**Comment by vincentyao** [comment]
@EngrPips  i am having the same issues. and in this page "https://github.com/smartcontractkit" I can't find  smartcontractkit/chainlink-brownie-contracts, so I need to run "forge install  https://github.com/smartcontractkit/chainlink --no-commit"?

**Reply by EngrPips** [reply]
Did the suggestion marked as an answer not work for you?

**Reply by vincentyao** [reply]
Yes, because as I mentioned earlier,  there is no smartcontractkit/chainlink-brownie-contracts repo now, but I have already fixed it. thanks for your reply

**Reply by EngrPips** [reply]
> Yes, because as I mentioned earlier, there is no smartcontractkit/chainlink-brownie-contracts repo now, but I have already fixed it. thanks for your reply

How did you fix it? If I may ask

**Reply by vincentyao** [reply]
https://github.com/Cyfrin/foundry-full-course-cu/discussions/3886#discussioncomment-12835667

**Reply by vincentyao** [reply]
@EngrPips
